frontend/public/locales/en/common.json

```diff
@@ -0,0 +1,196 @@
+    "deletePipelineFailed": "Failed to delete pipeline",
+    "deletePipelineVersionFailed": "Failed to delete pipeline version",
+    "deleteSelectedRuns": "Do you want to delete the selected runs? This action cannot be undone.",
+    "deleteSomePipelinesFailed": "Failed to delete some pipelines and/or some pipeline versions",
+    "deleteTensorboard": "Delete Tensorboard",
+    "deletionSucceeded": "Deletion succeeded for",
+    "description": "Description",
+    "descriptionOptional": "Description (optional)",
+    "details": "details",
+    "disable": "Disable",
+    "disabled": "Disabled",
+    "disableRunsTrigger": "Disable the run's trigger",
+    "dismiss": "Dismiss",
+    "documentation": "Documentation",
+    "dropFiles": "Drop files..",
+    "duration": "Duration",
+    "enable": "Enable",
+    "enabled": "Enabled",
+    "enableRunsTrigger": "Enable the run's trigger",
+    "endDate": "End date",
+    "endTime": "End time",
+    "error": "Error",
+    "errorInvalidEventVal": "Invalid event value type, expected string",
+    "errorOccurred": "An error occurred",
+    "errorRenderGraph": "There was an error rendering the graph.",
+    "errorResponse": "Error response",
+    "errorRetrieveResources": "Error retrieving resources",
+    "errorRunResource": "Error while running this resource",
+    "executePending": "Pending execution",
+    "executeResourceFailed": "Resource failed to execute",
+    "executions": "Executions",
+    "executionsList": "Executions List",
+    "executionSkipped": "Execution has been skipped for this resource",
+    "executionSkippedCache": "Execution was skipped and outputs were taken from cache",
+    "executionSuccess": "Executed successfully",
+    "expandCollapseSec": "Expand/Collapse this section",
+    "expandAll": "Expand all",
+    "expandAllSections": "Expand all sections",
+    "experiment": "Experiment",
+    "experimentList": "Experiment List",
+    "experiments": "Experiments",
+    "failedTo": "Failed to",
+    "file": "File",
+    "filter": "Filter",
+    "filterPipelines": "Filter pipelines",
+    "finishedAt": "Finished at",
+    "gettingStarted": "Getting Started",
+    "github": "Github",
+    "githubRepo": "Github Repo",
+    "graph": "Graph",
+    "graphDefInvalid": "Graph definition is invalid. Cannot get node by",
+    "here": "here",
+    "hide": "Hide",
+    "id": "ID",
+    "image": "Image",
+    "inputParams": "Input parameters",
+    "listRequestFailed": "List request failed with",
+    "listResourcesFailed": "Could not get requested list of resources",
+    "manage": "Manage",
+    "manifest": "Manifest",
+    "markdown": "Markdown",
+    "name": "Name",
+    "next": "Next",
+    "noPipelinesFound": "No pipelines found. Click \"Upload pipeline\" to start.",
+    "noPipelinesFoundTryAgain": "No pipelines found. Upload a pipeline and then try again.",
+    "noResourceFoundWithId": "Somehow no resource was found with ID",
+    "notebooks": "Notebooks",
+    "of": "of",
```

Comment:
This might be happening in more places, but translations such as these are hard to do because the 'sentence' might  have a different structure in other language. If possible, need to find a way to have one label, where you have attributes for the values.  (see NewRun.tsx comment)